### ISSUE TITLE
Folder icons in settings are hard to distinguish

### ISSUE THREAD

**Issue by maximilionus** [maximilionus/telegram-dark-shell#17]
## About
Icons in the **Settings -> Folders** sub-menu are hard to distinguish because of the low contrast. Colors should be changed to lighter values.

## Version
- Client: Telegram Desktop `4.8.7`
- Theme: `2.4`

## Screenshot
![image](https://github.com/maximilionus/telegram-dark-shell/assets/29492157/51160d8d-9be1-403e-9b93-7ad07d10f2a4)

**Comment by elifian** [comment]
Unfortunately, it's a double-edged sword. The color index is tied to the buttons, so changing the color also affects all the buttons, which negatively impacts the overall appearance.

**Comment by maximilionus** [comment]
@elifian exactly. This is the reason why this issue is still not fixed. I was hoping desktop client will eventually get this elements split from the rest, but that never happened.
